server/GraknServer.java

```diff
@@ -279,49 +241,24 @@ private static void startGraknServer(ServerCommand.Start command) throws IOExcep
         LOG.info("- listening to port: {}", server.port());
         LOG.info("- data directory configured to: {}", server.dataDir());
         LOG.info("- bootup completed in: {} ms", Duration.between(start, end).toMillis());
-        LOG.info("");
-        LOG.info("Grakn Core Server is now running and will keep this process alive.");
-        LOG.info("You can press CTRL+C to shutdown this server.");
         LOG.info("...");
         server.serve();
     }
 
-    @Override
-    public void close() {
-        LOG.info("");
-        LOG.info("Shutting down Grakn Core Server...");
-        try {
-            graknService.close();
-            server.shutdown();
-            server.awaitTermination();
-            grakn.close();
-            System.runFinalization();
-            LOG.info("Grakn Core Server has been shutdown");
-        } catch (InterruptedException e) {
-            LOG.error(FAILED_AT_STOPPING.message(), e);
-            Thread.currentThread().interrupt();
-        }
+    protected static void printSchema(ServerCommand.PrintSchema printSchemaCommand) {
```

Comment:
I don't think it makes sense for us to extract the migrator-related method since they're actually quite small. What do you think Haikal?